smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/DocumentMemberSerVisitor.java

```diff
@@ -58,16 +60,17 @@
  *   <li>Timestamp: converted to a representation based on the specified format.</li>
  *   <li>Service, Operation, Resource, Member: not serializable in documents. <b>Not overridable.</b></li>
  *   <li>Document, List, Map, Set, Structure, Union: delegated to a serialization function.
- *     <b>Not overridable.</b></li>
+ *     <b>Not overridable.</b> These delegations respect the {@code additionalParams} property.</li>
```

Comment:
Is there more documentation somewhere about what this actually means? i.e., "Any provided additionalParams will appear as insertion ordered trailing arguments in the generated serializer method."